tests/potential/test_pair.py

```diff
@@ -0,0 +1,147 @@
+"""Unit tests for pair module."""
+import unittest
+
+import numpy as np
+
+from relentless import core
+from relentless.potential import pair
+from relentless.potential import potential
+
+class test_LennardJones(unittest.TestCase):
+    """Unit tests for pair.LennardJones"""
+
+    def test_init(self):
+        """Test creation from data"""
+        lj = pair.LennardJones(types=('1','2'))
+        coeff = potential.CoefficientMatrix(types=('1','2'), params=('epsilon','sigma','rmin','rmax','shift'),
+                                            default={'rmin':False,'rmax':False,'shift':False})
+        self.assertCountEqual(lj.coeff.types, coeff.types)
+        self.assertCountEqual(lj.coeff.params, coeff.params)
+        self.assertDictEqual(lj.coeff.default.todict(), coeff.default.todict())
+
+    def test_accessors(self):
+        """Test accessor methods for coeffs"""
+        lj = pair.LennardJones(types=('1','2'))
+
+        #test update with kwargs
+        lj.coeff['1','1'].update(epsilon=2.0, sigma=1.5)
+        self.assertAlmostEqual(lj.coeff['1','1']['epsilon'], 2.0)
+        self.assertAlmostEqual(lj.coeff['1','1']['sigma'], 1.5)
+        self.assertEqual(lj.coeff['1','2']['epsilon'], None)
+        self.assertEqual(lj.coeff['1','2']['sigma'], None)
+
+        #test update with positional arg
+        lj.coeff['1','2'].update({'epsilon':2.5})
+        self.assertAlmostEqual(lj.coeff['1','1']['epsilon'], 2.0)
+        self.assertAlmostEqual(lj.coeff['1','1']['sigma'], 1.5)
+        self.assertAlmostEqual(lj.coeff['1','2']['epsilon'], 2.5)
+        self.assertEqual(lj.coeff['1','2']['sigma'], None)
+
+        #test setitem with default reset
+        lj.coeff['1','2'] = {'sigma':0.5}
+        self.assertAlmostEqual(lj.coeff['1','1']['epsilon'], 2.0)
+        self.assertAlmostEqual(lj.coeff['1','1']['sigma'], 1.5)
+        self.assertEqual(lj.coeff['1','2']['epsilon'], None)
+        self.assertAlmostEqual(lj.coeff['1','2']['sigma'], 0.5)
```

Comment:
You can leave this test if you want (since it's already written), but it is probably not necessary because you already have good coverage of the `CoefficientMatrix` elsewhere and you've already checked that `lj.coeff` is getting the right parameters in `test_init`.